pkg/cleaner/aws/aws.go

```diff
@@ -68,24 +68,37 @@ func (a *Cleaner) cleanStacks() error {
 		return microerror.Mask(err)
 	}
 
+	var lastError error
 	for _, stack := range output.Stacks {
 		a.logger.Log("level", "debug", "message", fmt.Sprintf("checking stack %#q", *stack.StackName))
 		if !stackShouldBeDeleted(stack) {
 			continue
 		}
 		a.logger.Log("level", "debug", "message", fmt.Sprintf("found that stack %#q should be deleted", *stack.StackName))
+
+		enableTerminationProtection := false
+		updateTerminationProtection := &cloudformation.UpdateTerminationProtectionInput{
+			EnableTerminationProtection: &enableTerminationProtection,
+			StackName:                   stack.StackName,
+		}
+		_, err = a.cfClient.UpdateTerminationProtection(updateTerminationProtection)
+		if err != nil {
+			return microerror.Mask(err)
+		}
```

Comment:
Do I keep the same behavior than delete and save the error but continue the loop?

Reply:
I think we should save lastError here too, same as below.